integration/boofcv-javacv/src/main/java/boofcv/javacv/ConvertIplImage.java

```diff
@@ -264,21 +265,14 @@ private static void convertFrom_I( IplImage input, InterleavedF64 output ) {
 	}
 
 	public static ImageDataType depthToBoofType( int depth ) {
-		switch (depth) {
-			case IPL_DEPTH_8U:
-				return ImageDataType.U8;
-			case IPL_DEPTH_8S:
-				return ImageDataType.S8;
-			case IPL_DEPTH_16S:
-				return ImageDataType.S16;
-			case IPL_DEPTH_32S:
-				return ImageDataType.S32;
-			case IPL_DEPTH_32F:
-				return ImageDataType.F32;
-			case IPL_DEPTH_64F:
-				return ImageDataType.F64;
-			default:
-				throw new IllegalArgumentException("Unknown IPL depth " + depth);
-		}
+		return switch (depth) {
```

Comment:
In the future, try to do these changes in a seperate PR. Besides easier to review, it's easier to debug in the future as you can see the functional changes. Yes I should be more strict about this myself.

Reply:
Ahh, I missed removing this one. I'll pull that out.